astropy/io/fits/hdu/hdulist.py

```diff
@@ -94,6 +98,12 @@ def fitsopen(name, mode='readonly', memmap=None, save_backup=False,
         Do not issue an exception when opening a file that is missing an
         ``END`` card in the last header. Default is `False`.
 
+    ignore_missing_simple : bool, optional
+        Do not issue an exception when the SIMPLE keyword is missing.
```

Comment:
```suggestion
        Do not raise an exception when opening a file that is missing an
        ``END`` card in the last header. Default is `False`.

    ignore_missing_simple : bool, optional
        Do not raise an exception when the SIMPLE keyword is missing.
```
Nitpick: I think `raise` is the standard terminology; not sure if there are official recommendations, but to be that verb makes it clearer that program execution will be interrupted.

Reply:
Indeed, `raise` seems better. I did the change locally since I needed to rebase and move the changelog entry, to avoid running the CI twice.